src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -237,8 +266,12 @@ public int compareTo(ReportItem other) {
   Map<String, ReportItem> allRna = Maps.newTreeMap();
   List<Map.Entry<String, ReportItem>> dnaRecentList;
   List<Map.Entry<String, ReportItem>> rnaRecentList;
+  List<Map.Entry<String, ReportItem>> dnaStagnantList;
+  List<Map.Entry<String, ReportItem>> rnaStagnantList;
   List<Map.Entry<String, ReportItem>> dnaAllList;
   List<Map.Entry<String, ReportItem>> rnaAllList;
+  private Map<Integer, InstrumentDto> instrumentsById;
+  private Map<Integer, InstrumentModelDto> instrumentModelsById;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "instrumentModelsById" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)